rescpu.c

```diff
@@ -29,9 +29,7 @@
 #include <libgen.h>
 #include <ctype.h>
 
-//#define TESTING
-
-static int populate_cpuinfo(void *out, char *buffer, int total_len)
+static int populate_cpuinfo(void *out, char *buffer, int total_len, int ex)
```

Comment:
maybe rename 'ex' to 'probe' or something (and invert it)?  And then add a comment above explaining that if probe is true then you only set the value to 1 to say it was present.